app/helpers/mailer_helper.rb

```diff
@@ -176,6 +191,85 @@ def style_work_tag_metadata(tags)
     "#{label}#{style_work_tag_metadata_list(tags)}".html_safe
   end
 
+  # The subject of batch_subscription_notification is based on the first
+  # creation in the email. It uses that creation's creator, so if you subscribe
+  # to user X, who has co-created a work with user Y, and Y posts a chapter that
+  # is not co-created, the subject line will say just "Y posted" even though the
+  # subscription is for X.
+  # Note that we assume this is being used in batch_subscription_notification,
+  # after a subscription.valid_notification_entry?(creation) check. You can
+  # otherwise pass invalid or anonymity-breaking combinations of data to this.
+  def batch_subscription_subject(subscription, creation, additional_creations_count)
+    subscribable_type = subscription.subscribable_type.downcase
+
+    creation_type = creation.is_a?(Chapter) ? "chapter" : "work"
```

Comment:
You can simplify this by calling `creation.model_name.i18n_key`, here and elsewhere.